docs/ide/set_up_vscode.md

```diff
@@ -215,6 +215,62 @@ Navigate to DataCatalog:
 Clicking on a data node in the flowchart will open the corresponding dataset in the Data Catalog.
 ![navigation to dataset](../meta/images/viz-vsc-nav-data-node.gif)
 
+## Auto reload Kedro-Viz
+
+The extension provides an auto reload feature that automatically refreshes Kedro-Viz when you make changes to your Kedro project files.
+
+### What is auto reload?
+
+When enabled, the auto reload feature monitors your Kedro project files for changes. If the Kedro-Viz panel is open and you change any monitored files, the extension automatically restarts the Kedro server and refreshes the visualisation.
+
+### Monitored files
+
+The extension watches for changes in:
+- **Configuration files**: `**/conf/**/*.yml` and `**/conf/**/*.yaml`
```

Comment:
I don't think this rendered as points -- see the compiled docs; i think mkdocs for points is different style.